pyqtgraph/tests/test_functions.py

```diff
@@ -245,3 +245,18 @@ def test_siParse(s, suffix, expected):
     else:
         with pytest.raises(expected):
             pg.siParse(s, suffix=suffix)
+
+def test_CIELab_reconversion():
+    color_list = [ pg.Qt.QtGui.QColor('#100235') ] # known problematic values
+    for _ in range(20):
+        qcol = pg.Qt.QtGui.QColor()
+        qcol.setRgbF( *np.random.random((3)) )
+        color_list.append(qcol)
+    
+    for qcol1 in color_list:
+        vec_Lab  = pg.functions.colorCIELab( qcol1 )
+        qcol2 = pg.functions.CIELabColor(*vec_Lab)
+        for val1, val2 in zip( qcol1.getRgb(), qcol2.getRgb() ):
+            if abs(val1-val2) > 1:
+                print(f'Excess CIELab reconversion error ({qcol1.name() } > {vec_Lab } > {qcol2.name()})')
```

Comment:
move this print-statement into the assert error message

```python
assert False, f"Excess CIELab ....."
```

This way the test suite won't capture that output, but it will be displayed on the test summary.

Reply:
Today I learn... Thanks, added.